### ISSUE TITLE
Feature request: display update progress in web interface

### ISSUE THREAD

**Issue by barcoboy** [ampache/ampache#2786]
Would it be possible to display the progress of "git pull https://github.com/ampache/ampache.git develop" and "composer install --prefer-source --no-interaction" when the "Update" link is clicked in the web interface?

This morning when updating one of my installations, there was a local change in one of my files that was preventing git from doing the pull, and the web interface would immediate return back and still say that an update was available, not showing me that there was an issue.  To see what the problem was, I had to ssh into my server and run the git pull process manually.

Just updating another installation now, and after clicking on the update link, the page timed out after about a minute while the update continued on in the background (it did go on in the background to complete successfully).

**Comment by barcoboy** [comment]
If implemented, a click to return link would probably be a good idea so that if an update takes a long time and the user switches tabs or does something else, the results remain on the screen until the user clicks the link to go back.

**Comment by lachlan-00** [comment]
I think a message of success or fail could be done instead of reloading the index page

**Comment by kuzi-moto** [comment]
@barcoboy, I looked into this a while ago and I'm pretty certain that page just outputs that text to the screen and runs the command in the background. No matter the result, the same text is displayed each time.

I think the updating experience could be a little nicer if the page was styled in the same way as the rest of the interface as well.

**Comment by lachlan-00** [comment]
yeah that's true, i might make up and update page for it. then when you click on update it takes you there and lest you know what actually happened.

**Comment by lachlan-00** [comment]
we can't easily show progress but with ampache 8 any failure will stop and display the progress and the error so it's a lot more robust now when it fails
